### PR TITLE
Clean up unused structures

### PR DESCRIPTION
**Pull request by Fish-pro**
Signed-off-by: Fish-pro <zechun.chen@daocloud.io>

Clean up unused structures and adjust the order of imported packages

### CODE REVIEW

File: pkg/types/types.go [hunk 1]
```diff
@@ -16,11 +16,11 @@
 package types
 
 import (
-	"gopkg.in/k8snetworkplumbingwg/multus-cni.v3/pkg/logging"
```
Comment (s1061123) [inline pkg/types/types.go:19]: Could you please let me know why you change the import list? If you don't have any clear reason, we'd prefer to keep as it is.

**Review by s1061123 — CHANGES_REQUESTED** [review]

File: pkg/types/types.go (hunk 1 above)
Comment (Fish-pro) [reply pkg/types/types.go:19]: @s1061123 The package import order is confusing. For example, `golangci-lint` has an explicit requirement for package import order. Also, the package import order here is different from the other code files. Typically, golang's built-in packages are imported first, and then third-party packages are imported

Comment (s1061123) [reply pkg/types/types.go:19]: Currently, as far as I know of, we don't have strict import rule in this repository yet. So this change should be done once we have the one. Please revert this change for now.

Comment (Fish-pro) [reply pkg/types/types.go]: Order has been preserved

**Pull request closed without merging**